### DISCUSSION TITLE
Debug connection for Microchip/Atmel SAME54 Xplained Pro evaluation board.

### DISCUSSION THREAD

**Discussion by MyGh64605** [zephyrproject-rtos/zephyr#77762] [Q&A]
What are steps required to connect and debug using this board ?

**Comment by henrikbrixandersen** [comment]
Have you tried the instructions from here? https://docs.zephyrproject.org/latest/boards/atmel/sam0/same54_xpro/doc/index.html#programming-and-debugging

**Reply by MyGh64605** [reply]
Thank you for your reply.
The 'opencfg' in board.cmake for the SAME54 has the contents below.

Where do I find the following ?
source [find interface/cmsis-dap.cfg]
source [find target/atsame5x.cfg]

>>>>>>
# Copyright (c) 2018 Bryan O'Donoghue
# SPDX-License-Identifier: Apache-2.0
source [find interface/cmsis-dap.cfg]
transport select swd

# chip name
set CHIPNAME atsame54p20a
set ENDIAN little
set CPUTAPID 0x0bc11477

source [find target/atsame5x.cfg]

reset_config none
cortex_m reset_config sysresetreq

$_TARGETNAME configure -event gdb-attach {
	echo "Debugger attaching: halting execution"
	reset halt
	gdb_breakpoint_override hard
}

$_TARGETNAME configure -event gdb-detach {
	echo "Debugger detaching: resuming execution"
	resume
}
<<<<<<<
